softdevice_controller/CHANGELOG.rst

```diff
@@ -35,7 +35,7 @@ Changes
   Generally, this change will result in either increased scanning time, or similar scanning time to before.
   In cases where there are many conflicting activities within the scan window, this change may result in reduced scanning time. (DRGN-19050)
 * Scan windows are no longer limited to 16 seconds. (DRGN-19050)
-* The deprecated functions :c:func:`rand_prio_low_get` and :c:func:`rand_prio_high_get` have been removed.
+* The deprecated  :c:member:`sdc_rand_source_t.rand_prio_low_get` and :c:member:`sdc_rand_source_t.rand_prio_high_get` have been removed.
```

Comment:
```suggestion
* The deprecated :c:member:`sdc_rand_source_t.rand_prio_low_get` and :c:member:`sdc_rand_source_t.rand_prio_high_get` have been removed.
```

Reply:
Now that these structure members are removed, should we just have them between double backticks instead of the current formatting?